### ISSUE TITLE
Email auth docs don't explain how to use `signup` and `login` actions

### ISSUE THREAD

**Issue by infomiho** [wasp-lang/wasp#1438]
We want to add something similar to what we did for Username & password auth: https://wasp-lang.dev/docs/auth/username-and-pass#customizing-the-auth-flow

**Comment by Young-Einstein10** [comment]
@infomiho I'm interested in working on this. 

If I get the task correctly, we're trying to have an example in the docs similar to what's [here](https://wasp-lang.dev/docs/auth/username-and-pass#customizing-the-auth-flow) for the email section. i.e., How to use the `signup` and `login` actions, right?

**Comment by infomiho** [comment]
Yes, that's the idea. Add a part that explains how to use custom UI and not Wasp's provided Auth UI.

**Comment by Young-Einstein10** [comment]
Alright. I'll work on this and open a draft PR soon.

**Comment by Young-Einstein10** [comment]
Hey @infomiho 
I still plan on fixing this. I'm going through the Project Setup [here](https://github.com/wasp-lang/wasp/tree/main/waspc) so I can set things up on my machine. I'll let you know if get any blockers.

**Comment by Young-Einstein10** [comment]
Hey @infomiho 
I think I'll be dropping this issue for now. I'm experiencing errors setting up the project on my machine and don't have much time to debug in-depth. I'll join the discord channel later this week. Hopefully, someone might be able to help in there.

**Comment by Martinsos** [comment]
> Hey @infomiho I think I'll be dropping this issue for now. I'm experiencing errors setting up the project on my machine and don't have much time to debug in-depth. I'll join the discord channel later this week. Hopefully, someone might be able to help in there.

Sounds good, we will be happy to help!

**Comment by Martinsos** [comment]
Hey @n16htb0t , sure go for it! You do have `bot` in your name, I do hope you are not a bot though :)?

**Comment by Martinsos** [comment]
Another user hitting this: https://discord.com/channels/686873244791210014/1313684793288032267/1325525326339510306 .

**Comment by Martinsos** [comment]
I am also thinking we would benefit from a central place that describes how to customize Auth Forms. Right now the approach is that we describe it under each auth provider, but that is hard to find. And naming is also a bit hard to figure out: under email and pass, it is "using login and signup action", but it would make more sense if name was "implementing your own Login / Signup form" because that is what people are looking for. So yeah, better title for that part, plus a central place in the Auth docs where we describe the "onion": You can use our forms as they are, you can use our forms + customize them via their props, or you can impelment your forms but use our actions, or even go furhter and use your forms and your actions, but our backend stuff and database entities. So 3-4 layers. We could describe that in central place, and then from there link to specific for each method (or tell them to look under each method/provider).

**Comment by Martinsos** [comment]
Related to https://github.com/wasp-lang/wasp/issues/1710 .

**Comment by cprecioso** [comment]
> Right now the approach is that we describe it under each auth provider, but that is hard to find.

I think this is something that will be easy to solve once we have an API reference (#1401), because we can just describe it centrally there and link it from the relevant places.
